pkg/rpm/salt-minion

```diff
@@ -25,12 +25,11 @@
 # processname: /usr/bin/salt-minion
 
 # Allow these to be overridden for tests
-: ${SALTMINION_BINDIR:=/usr/bin}
-: ${SALTMINION_SYSCONFDIR:=/etc}
-: ${SALTMINION_PYTHON:=python}    # Python must be 2.6 or newer
```

Comment:
Nevermind, that would break minions that have 2.7 and still use sysvinit.

How about simply deploying an ``/etc/default/salt`` with the EL5 package, which sets ``SALTMINION_PYTHON=python2.6``?

Actually, since the other initscripts all use a ``PYTHON`` shell variable, we should probably set ``PYTHON=python2.6`` in an ``/etc/default/salt`` that we deploy for EL5, and remove the ``SALTMINION_`` from all variables in the initscript.